### ISSUE TITLE
Doesnt work

### ISSUE THREAD

**Issue by Helenvog** [buddydev/bp-live-notification-tone#1]
I really need plugin like this on my website...but it doesnt work...Any idea?

**Comment by sbrajesh** [comment]
do you use any plugin that disables WordPress heartbeat api?

**Comment by Helenvog** [comment]
Thanks for quick response:-) Well,i am not sure...here is list of plugins what i use: Admin Custom Login,Advanced iFrame,Akismet,bbPress,Bp Favorite Notification,BP Login Redirect,BP Profile as Homepage,BP Reshare,BP Simple Front End Post,BuddyBlog,BuddyBoss Wall,BuddyPress,BuddyPress Activity Plus,BuddyPress Global Search,BuddyPress TimeZones,iFlyChat,Like Button Voting & Rating,Rename wp-login.php,rtMedia for WordPress, BuddyPress and bbPress,Wordpress Video Gallery,WP Job Manager.

**Comment by sbrajesh** [comment]
Hi,
Do you have this plugin installed?
https://wordpress.org/plugins/bp-live-notification/

**Comment by Helenvog** [comment]
No...I had it but it also didnt work and i removed it from plugins...By the way i am using Boss theme...
